plugin/audit_log/filter.c

```diff
@@ -167,13 +263,26 @@ void audit_log_filter_init()
                20, 0, 0,
                (my_hash_get_key) account_get_key,
                my_free, HASH_UNIQUE, key_memory_audit_log_accounts);
+
+  my_hash_init(&include_databases, &my_charset_utf8_general_ci,
```

Comment:
I think here (and for account filter hashes) there is character set collation overhead on each and every lookup? Would it be possible to create hashes with one of the dumb character sets, and convert to it on insert to hash?